takoyaki/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +37,17 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+    // Run Intake
+    new JoystickButton(driveController, Constants.kHooper.RUN_HOPPER)
+      .whenPressed(new InstantCommand(hopper::runForward, hopper))
+      .whenReleased(new InstantCommand(hopper::stop, hopper));
+
+    // Reserve Intake
```

Comment:
nit: spelling: reverse instead of reserve
```suggestion
    // Reverse Intake
```